components/object-property-list/object-property-list.js

```diff
@@ -50,10 +53,23 @@ class ObjectPropertyList extends LocalizeCoreElement(SkeletonMixin(LitElement))
 			<div class="d2l-body-small">
 				<slot name="status"></slot>
 				<d2l-screen-reader-pause></d2l-screen-reader-pause>
-				<slot>${slotContents}</slot>
+				<slot @slotchange="${this._onItemsChanged}">${slotContents}</slot>
 			</div>
 		`;
 	}
+
+	_onItemsChanged() {
+		this._setItemSeparatorVisibility();
+	}
+
+	_setItemSeparatorVisibility() {
+		const slottedElements = this._slot.assignedElements();
```

Comment:
It looks like we're assuming that the `slot` element will never change. To avoid the assumption, we could query for the slot when needed, but in this particular case, we can have it passed in...
```javascript
_onItemsChanged(e) {
	this._setItemSeparatorVisibility(e.target); //e.target of slotchange is the slot element
}
_setItemSeparatorVisibility(slot) {
	...
}
```